data/tabular/rdkit_features/meta.yaml

```
@@ -0,0 +1,151 @@
+---
+names: rdkit_features
+description: |-
+    Molecular descriptors computed using RDKit
+targets:
+    - id: formula
+      type: formula
+      names:
+          - noun: formula
+          - noun: chemical formula
+          - noun: chemical formula
+          - noun: molecular formula
+    - id: NumHDonors
+      type: continuos
```

```suggestion
      type: continuous
```
this needs also replacement below